### DISCUSSION TITLE
Anyone at 38C3?

### DISCUSSION THREAD

**Discussion by vroland** [vroland/epdiy#388] [General]
Hi, 

if anyone is at 38C3 (https://events.ccc.de/congress/2024/infos/index.html), and wants to meet up to talk about e-paper stuff or other fun things, let me know! (Either this thread or DECT [88566](tel:88566). 
If you're not familiar with the event: Check out https://media.ccc.de, you'll probably find some talks you're interested in.

**Comment by martinberlin** [comment]
Hey Valentin thanks for posting this next time please post also the event location since is not clear in this page and it seems to have multiple locations.
Please tell me if one day you want to visit sunny Barcelona
